src/api/app/views/event_mailer/comment_for_request.html.haml

```diff
@@ -1,5 +1,7 @@
+- url = url_for(controller: 'webui/request', action: :show, number: event['number'], host: @host, only_path: false)
+
 %p
-  Visit #{link_to(*[url_for(controller: 'webui/request', action: :show, number: event['number'], host: @host, only_path: false)]*2)}
+  Visit #{link_to(*[url] * 2)}
```

Comment:
that's a trick that actually do not make the code better. That's just to make the linter happy.

Reply:
I found a better solution, please have a look.